scripts/classification/imagenet/verify_pretrained.py

```diff
@@ -56,12 +59,19 @@
 
 normalize = transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
 
-transform_test = transforms.Compose([
-    transforms.Resize(256, keep_ratio=True),
-    transforms.CenterCrop(224),
-    transforms.ToTensor(),
-    normalize
-])
+if 'inceptionv3' in model_name:
+    transform_test = transforms.Compose([
+        transforms.CenterCrop(299),
```

Comment:
Both `CenterCrop` should take in `input_size`.

Reply:
Thanks for the review, modification done accordingly.